### PR TITLE
NT-963 - No supported browser page

### PR DESCRIPTION
**Pull request by eiriko2303**

### CODE REVIEW

File: src/pages/not-supported-browser/not-supported-browser.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+not-supported-browser {
+    .nosupport {
+        &-content, &-footer {
+            &-image, &-text {
+                display: block;
+                margin-left: auto;
+                margin-right: auto;
```
Comment (hasnik) [inline src/pages/not-supported-browser/not-supported-browser.scss]: I believe _margin: auto;_ will be sufficient

Comment (hasnik) [reply src/pages/not-supported-browser/not-supported-browser.scss]: This site actually looks awful on mobile

File: src/app/app.component.ts [hunk 2]
```diff
@@ -16,6 +16,21 @@ import { setHasShownDownloadAlert } from '../actions/rootActions';
 import { AppVersion } from '@ionic-native/app-version';
 import { OneSignal } from '@ionic-native/onesignal';
 
+//Values that is browser specific
```
Comment (hasnik) [inline src/app/app.component.ts]: I believe the code for no supported browser should be enclosed in some other class, not in app.component

File: src/pages/not-supported-browser/not-supported-browser.scss (hunk 1 above)
Comment (eiriko2303) [reply src/pages/not-supported-browser/not-supported-browser.scss]: ok :)

File: src/app/app.component.ts (hunk 2 above)
Comment (eiriko2303) [reply src/app/app.component.ts]: How should it be used? It needs to be called first i think?

Comment (eiriko2303) [reply src/app/app.component.ts]: Ok i have placed the declaring of the variables in a seperate class, but the method needs to stay in app.component cause it needs to be available imediately at start of the app.

**Comment by hasnik** [comment]
I appreciate the changes, but as Kamil said. 'isSupportedBrowser' function should not belong to app.component

**Comment by eiriko2303** [comment]
Yes i understand, but the problem i got is how should i then have the function as a entry function when the app is loaded?

**Comment by reg0** [comment]
I fixed it 
@hasnik please re-review

**Comment by eiriko2303** [comment]
@reg0 Great work! thanks 😄

**Comment by hasnik** [comment]
@reg0 @eiriko2303 nice work!

**Pull request merged**